tests/credential-status/check.no-auth.spec.ts

```diff
@@ -0,0 +1,129 @@
+import {
+    TESTNET_DID,
+    CONTENT_TYPE,
+    PAYLOADS_PATH,
+    NOT_EXISTENT_TESTNET_DID,
+    NOT_EXISTENT_STATUS_LIST_NAME
+} from '../constants';
+import * as fs from 'fs';
+import { test, expect } from '@playwright/test';
+import { StatusCodes } from 'http-status-codes';
+
+test('[Positive] It can check an unencrypted status-list with an existent body and statusPurpose=revocation parameter', async ({ request }) => {
+    const response = await request.post('/credential-status/check?statusPurpose=revocation', {
+        data: {
+            did: TESTNET_DID,
+            index: 10,
+            statusListName: "cheqd-employee-credentials",
+        },
+        headers: { 'Content-Type': CONTENT_TYPE.APPLICATION_JSON }
+    });
+    expect(response.status()).toBe(StatusCodes.OK);
+
+    const body = await response.json();
+    const expected = JSON.parse(
```

Comment:
And also here. As @ankurdotb already mentioned, we need to check not all the fields but only specific list. I think it's better to use here some specific `expect` functions, like `expect_credential_status_list_unencrypted` which could check only significant fields

Reply:
Agreed, as said before, don't expect `strictEquals` to work since the body may change. You don't have to test for everything, but you _can _ test for things like whether the DID matches, the name matches, the type matches etc.